common-modules/src/main/java/com/softserve/if072/common/model/Store.java

```diff
@@ -11,6 +13,8 @@
 public class Store {
 
     private int id;
+
+    @NotEmpty(message = "{error.name.notnull}")
```

Comment:
@NotBlank will handle such case "       "
Plus add validation for size for each string field/ Size should be the same as in the DB